pkg/universe.dagger.io/alpha/kubernetes/helm/install.cue

```diff
@@ -5,10 +5,13 @@ import (
 	"universe.dagger.io/docker"
 )
 
+_#DefaultNamespace: "default"
```

Comment:
Nit: According to [convention](https://docs.dagger.io/1226/coding-style#private-names) that should be `_#defaultNamespace`